input/charts.csv

```diff
@@ -7,7 +7,7 @@ upcomingAuctions,answer,Upcoming auctions,What is the volume of auctions announc
 wholesaleDistortions,answer,Wholesale price distortions,Are fossil fuels generators benefiting from subsidies that affect the price at which they can sell electricity onto the wholesale market?,fundamentals,Wholesale price distortions,,,
 sectorWholesale,answer,Power sector wholesale,,fundamentals,Wholesale power market,,,
 currencyVariation,answer,Currency variation,How high is the ten year variation of the local currency against the US dollar?,fundamentals,Currency variation,,,
-offtakerRisk,answer,Offtaker risk,How high is the offtake risk for independent generators?,fundamentals,Offtaker risk,,,
+offtakerRisk,range,Offtaker risk,How high is the offtake risk for independent generators?,fundamentals,Offtaker risk,,,
```

Comment:
The type was changed, but the options were not.
They should be:
```
Offtaker risk,2,Very High
Offtaker risk,0,Very Low
```

Reply:
The scale is not linear, so keeping the current steps to see what we have to do here.

Reply:
Ok, let's get feedback